extension/src/plots/webview/contract.ts

```diff
@@ -134,8 +134,11 @@ export type CustomPlotsData = {
 export const isVegaPlot = (plot: Plot): plot is TemplatePlot =>
   plot.type === PlotsType.VEGA
 
-export type ImagePlot = ImagePlotOutput & {
+export type ImagePlot = Omit<ImagePlotOutput, 'boundingBoxes'> & {
   ind?: number
+  boundingBoxes?: {
+    [label: string]: BoundingBox[]
+  }
```

Comment:
[Q] Why not get data in this format from DVC? (We are being asked to define the shape of the API output)

Reply:
> [Q] Why not get data in this format from DVC? (We are being asked to define the shape of the API output)

Good point! That would reduce transforming on both Studio and VSCode's side. I'll mention that in my response tomorrow.

Reply:
Schema has been decided to be an object: 

```
{
  "boxes": {
    "cat": [
      { "left": 100, "right": 110, "top": 5, "bottom": 20 },
      { "left": 30, "right": 55, "top": 75, "bottom": 90 }
    ],
    "dog": [
      { "left": 80, "right": 100, "top": 25, "bottom": 50 }
    ]
  }
}
```